src/__tests__/MP4TagReader-test.js

```diff
@@ -167,3 +167,31 @@ describe("MP4TagReader", function() {
     });
   });
 });
+
+describe("MP4TagReader", function() {
```

Comment:
No need to repeat this, just add your new `pit` function call after line 196.
And put your custom mp4FileContents/mediaFileReader/tagReader variable logic inside the `pit` too.